css/_vertical_filmstrip_overrides.scss

```diff
@@ -8,6 +8,8 @@
         display: flex;
         flex-direction: column-reverse;
         height: 100%;
+        position: fixed;
+        z-index: $tooltipsZ;
```

Comment:
popper will set itself as position:fixed if any of its parents are position:fixed. This is necessary to get the InlineDialogs to pop out of the scrolling filmstrip.